test/monitoring_console/monitoring_console_test.go

```diff
@@ -57,11 +56,18 @@ var _ = Describe("Monitoring Console test", func() {
 				3. Wait for Monitoring Console status to go back to READY
 				4. Verify Standalone configured in Monitoring Console Config Map
 				5. Verify Monitoring Console Pod has correct peers in Peer List
+				--------------- RECONFIG WITH NEW MC --------------------------
+				6.  Reconfig S1 with 2nd Monitoring Console Name
+				7.  Check 2nd Monitoring Console Config Map to verify s1
+				8.  Deploy 2nd Monitoring Console Pod
+				9.  Verify Standalone pod is configured on Monitoring Console Pod
+				10. Verify 1st Monitoring Console Config Map is not configured with S1
+				11. Verify 1st Monitoring Console Pod is not configured with S1
 			*/
 
 			// Deploy Monitoring Console CRD
 			mc, err := deployment.DeployMonitoringConsole(deployment.GetName(), "")
-			Expect(err).To(Succeed(), "Unable to deploy Monitoring Console instance")
+			Expect(err).To(Succeed(), "Unable to deploy Monitoring Console One instance")
 
 			// Verify Monitoring Console is Ready and stays in ready state
 			testenv.VerifyMonitoringConsoleReady(deployment, deployment.GetName(), mc, testenvInstance)
```

Comment:
Old code but any reason we check MC state twice here?

Reply:
After we deploy Standalone MC goes into Updating Phase so we just wait until MC is ready before running additional verifications.